packages/now-cli/src/util/projects/link.ts

```diff
@@ -122,6 +123,22 @@ export async function linkFolderToProject(
     { encoding: 'utf8' }
   );
 
+  await writeFile(
+    join(path, NOW_FOLDER, NOW_FOLDER_README),
+    `> Why do I have a folder named ".now" in my project?
```

Comment:
Might be overkill, but I'm thinking it would be better to create this as a separate file, so that `node` doesn't have to parse it into memory every time `now` is run. Cleaner separation of concerns too.

Reply:
Where would you put the `README.txt` file in the tree?

Reply:
Maybe in `packages/now-cli/src/util/projects/NOW_DIR_README.txt`?

Reply:
I like it! Implemented in c7aa3b7